backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -297,18 +309,16 @@ export class ExperimentService {
     });
 
     experiment.partitions = experimentPartitions;
-    experiment.endDate = null;
-    experiment.startDate = null;
     experiment.endOn = null;
     experiment.createdAt = new Date();
     experiment.state = EXPERIMENT_STATE.INACTIVE;
     return this.create(experiment, user);
   }
 
-  private async updateExperimentSchedules(experimentId: string): Promise<void> {
-    const experiment = await this.experimentRepository.findByIds([experimentId]);
+  private async updateExperimentSchedules(experimentId: string,  entityManager?: EntityManager): Promise<void> {
+    const experiment = await entityManager.getRepository(Experiment).findByIds([experimentId]);
```

Comment:
Is this not throwing lint error?
entityManager is an optional parameter. What if it is null?  `entityManager.getRepository` will throw error

Reply:
Yes, need to also handle null value. I have updated the code.